src/actionlib/action_client.py

```diff
@@ -314,6 +314,8 @@ def __init__(self, action_goal, transition_cb, feedback_cb, send_goal_fn, send_c
         self.latest_goal_status = GoalStatus(status=GoalStatus.PENDING)
         self.latest_result = None
 
+        self.lost_goal_timestamp_delta = rospy.Duration(0.0)
```

Comment:
LOST goals come in many flavours, I think what this is doing is timing out pending goals that never got accepted.

So I would put forward `goal_ack_timeout` or `pending_goal_timeout`.